src/main/resources/com/sonyericsson/jenkins/plugins/bfa/CauseManagement/index.groovy

```diff
@@ -32,7 +32,7 @@ def f = namespace(lib.FormTagLib)
 def l = namespace(lib.LayoutTagLib)
 def j = namespace(lib.JenkinsTagLib)
 
-l.layout(permission: PluginImpl.UPDATE_PERMISSION) {
```

Comment:
Since VIEW_PERMISSION is inferred by UPDATE_PERMISSION you should be able to put ```permission: PluginImpl.VIEW_PERMISSION``` here for safety.